Bottom: d7a0eca0ff22a35d25206dfbb7372f6bd434699d
Top:    02e0dfebdd5e6d518dc03a23a9f370613e8ac548
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-04 17:47:27 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
